src/Nuget/Sarif.Multitool.nuspec

```diff
@@ -21,14 +21,14 @@
   </metadata>
   <files>
     <!-- The subfolder layout for the different TargetFrameworks is intentionally inconsistent -->
-    <!-- net472 and netcoreapp3.1 pack to tools\TargetFramework\* for backcompat -->
-    <file src="bld\bin\$platform$_$configuration$\Sarif.Multitool\Publish\net472\Sarif.Multitool.exe.config"
-          target="tools\net472" />
-    <file src="bld\bin\$platform$_$configuration$\Sarif.Multitool\Publish\net472\**"
-          target="tools\net472"
-          exclude="bld\bin\$platform$_$configuration$\Sarif.Multitool\Publish\net472\Sarif*;bld\bin\$platform$_$configuration$\Sarif.Multitool\Publish\net472\WorkItems.dll;" />
-    <file src="bld\bin\Signing\net472\**"
-          target="tools\net472" />
+    <!-- net462 and netcoreapp3.1 pack to tools\TargetFramework\* for backcompat -->
```

Comment:
Is this just to bring in line with our build settings for supported versions?

Reply:
this is to ensure that our own exes runs on the lowest possible fully supported .net fx.